Resources/Prototypes/Entities/Mobs/Player/admin_ghost.yml

```diff
@@ -95,6 +95,7 @@
   - type: InventorySlots
   - type: Loadout
     prototypes: [ MobAghostGear ]
+  - type: SupermatterImmune
```

Comment:
Shouldn't normal ghosts have this instead?

Reply:
Unnecessary, regular ghosts don't have hands or collision, they can't actually touch the Supermatter.